content/en/database_monitoring/setup_oracle/rds.md

```diff
@@ -55,41 +63,52 @@ exec rdsadmin.rdsadmin_util.grant_sys_object('V_$SGAINFO','DATADOG','SELECT',p_g
 exec rdsadmin.rdsadmin_util.grant_sys_object('V_$SYSMETRIC','DATADOG','SELECT',p_grant_option => false);
 ```
 
-## Install the Agent
+## Configure the Agent
 
-To start collecting Oracle telemetry, first [install the Datadog Agent][1]. Note you must install the [correct version][7] to participate in the beta. 
+To start collecting Oracle telemetry, first [install the Datadog Agent][1]. 
 
 Create the Oracle Agent conf file `/etc/datadog-agent/conf.d/oracle-dbm.d/conf.yaml`. See the [sample conf file][2] for all available configuration options.
 
 ```yaml
 init_config:
 instances:
-  - dbm: true
-    server: '<RDS_INSTANCE_ENDPOINT>:<SQL_PORT>'
+  - server: '<RDS_INSTANCE_ENDPOINT_1>:<PORT>'
     service_name: "<SERVICE_NAME>" # The Oracle CDB service name
     username: 'c##datadog'
     password: '<PASSWORD>'
+    reported_hostname: '<USER_FRIENDLY_HOSTNAME>' #Optional
```

Comment:
I think we should remove this. This is actually not recommended for RDS because we want users to connect to the instance endpoint directly (and should show up that way in the UI)